clients/drcachesim/tests/scheduler_launcher.cpp

```diff
@@ -177,11 +183,13 @@ simulate_core(int ordinal, scheduler_t::stream_t *stream, const scheduler_t &sch
             std::cerr << line.str();
         }
         scheduler_t::input_ordinal_t input = stream->get_input_stream_ordinal();
-        if (thread_sequence.empty())
-            thread_sequence.push_back(input);
-        else if (stream->get_input_stream_ordinal() != prev_input) {
-            thread_sequence.push_back(input);
-            if (op_verbose.get_value() > 0) {
+        if (input != prev_input) {
+            // We convert to letters which only works well for <=26 inputs.
```

Comment:
Well, we could use A-Z,a-z,0-9, but maybe that's not worth it as even that could be exceeded.

Reply:
Agreed, for upstream use as-is seems enough.  Even for thousand-thread-input testing the letters still give a good visual idea of how much code is run between context switches.